.dockerignore

```diff
@@ -0,0 +1 @@
+.gitignore
```

Comment:
Same as the dockerfile.

Reply:
I'm curious about the propose of this file. Why should be ignore `.gitignore` when building docker image?